openml/runs/functions.py

```diff
@@ -906,6 +905,14 @@ def obtain_field(xml_obj, fieldname, from_server, cast=None):
 
     tags = openml.utils.extract_xml_tags("oml:tag", run)
 
+    # Make sure default values are used where needed to keep run objects identical
+    if not evaluations:
+        evaluations = None
+    if not fold_evaluations:
+        fold_evaluations = None
+    if not sample_evaluations:
+        sample_evaluations = None
```

Comment:
Could instead pass the parameter value `evaluations or None` which means that it's none with an empty dict, and `evaluations` otherwise.

Reply:
Indeed, good suggestion, I updated this. Looks much cleaner.